documentation/modules/configuring/proc-scaling-up-a-kafka-cluster.adoc

```diff
@@ -4,12 +4,12 @@
 
 [id='proc-scaling-up-a-kafka-cluster-{context}']
 
-= Scaling up a Kafka cluster
+= Reassigning partitions after adding brokers
```

Comment:
Same as above. Didn't we wanted to have the Cruise cotnrol mentioned here?

Reply:
I've added a note